scripts/migration/migrate_personal_to_profile_websites.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate single value personal to profile_websites list."""
+
+import logging
+
+from modularodm import Q
+from website import models
+from website.app import init_app
+
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    init_app(routes=False)
+    logger.info("migrating personal to profileWebsites")
+    all_users = 0
+    migrated_users = 0
+    for user in get_users_with_social_field():
+        all_users += 1
+        logger.info("Migrating User: %s" % repr(user.fullname))
```

Comment:
Actually, this logging statement is inaccurate and unnecessary. This will run for every user that has *any* social field set. This should only be run for users that get migrated.